avo-hq/avo
# feature: pills switcher

# Description


Fixes https://github.com/avo-hq/avo/pull/1137

[Docs](https://docs.avohq.io/2.0/tabs.html#display-as-pills)

![CleanShot 2022-08-22 at 10 42 58@2x](https://user-images.githubusercontent.com/1334409/185867062-3b871a32-78f9-4619-bf19-68e30d19b765.jpg)


https://user-images.githubusercontent.com/1334409/185878771-84ca879e-e0f7-4a05-8cbc-de61bb84fbd4.mp4




# Checklist:


- [ ] Implement design from designer


## Manual review steps


1. Go to a fish
1. Notice the tab switcher displayed as pills

Manual reviewer: please leave a comment with output from the test if that's the case.
